travis-ymls/mothur.travis.yml

```diff
@@ -0,0 +1,36 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : mothur
+# Source Repo         : https://github.com/ujjwalsh/mothur
```

Comment:
Please add main repo link and not your fork.

Reply:
Done